### ISSUE TITLE
encodeAppendSequencerBatch doesn't include method ID

### ISSUE THREAD

**Issue by smartcontracts** [ethereum-optimism/core-utils#6]
`encodeAppendSequencerBatch` correctly encodes everything except the method ID. Clients of the library end up having to manually add the method ID everywhere. I think it's easier to return with the method ID and allow clients to slice of the first four bytes if they want.

https://github.com/ethereum-optimism/core-utils/blob/98ef6e0f6b4b237c894b95b7e00153f889819e80/src/coders/sequencer-batch.ts#L40-L66

Fix here would be to modify the return value to start with:

```ts
`ethers.utils.id('appendSequencerBatch()').slice(0, 10)` // just want 0x + first four bytes = 10 total characters
```

**Comment by platocrat** [comment]
could use a review on my PR

**Comment by platocrat** [comment]
This issue has also been patched in #8 

https://github.com/ethereum-optimism/core-utils/blob/8c5a674acb902b8f8c288104308272970ae11431/src/coders/sequencer-batch.ts#L128-L146

**Comment by tynes** [comment]
Thanks for the help @platocrat !
